helpers.go

```diff
@@ -32,5 +32,9 @@ func IntToHex(i int) string {
 
 // BigToHex covert big.Int to hexadecimal representation
 func BigToHex(bigInt big.Int) string {
-	return "0x" + strings.TrimPrefix(fmt.Sprintf("%x", bigInt.Bytes()), "0")
+	if bigInt.Cmp(big.NewInt(0)) == 0 {
```

Comment:
Just use [BitLen](https://golang.org/pkg/math/big/#Int.BitLen) method for zero checking.